### DISCUSSION TITLE
Postman request to the .net SDK

### DISCUSSION THREAD

**Discussion by edwinkirubanithi** [FirelyTeam/firely-net-sdk#2681] [Q&A] [answered]
How do I translate this postman request to the .net SDK [https://server.fire.ly/r4/Encounter?patient=P23JOH, not able to find any options.

**Comment by mmsmits** [comment, marked as answer]
Hi @edwinkirubanithi .

You can use our FHIR Client class for this. 

You can find documentation about search using the FHIR client here:

https://docs.fire.ly/projects/Firely-NET-SDK/en/latest/client/search.html
